CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # credentials-context ChangeLog
 
+## 1.1.0 - 
```

Comment:
Is this really a non-breaking change? If that's the assertion (and its not a major one), then it needs to be symlinked and tested in applications that are using 1.0 today.

Reply:
+1 to testing, but I'm also happy to make it a breaking change.

Reply:
@dlongley - changed to breaking change.